spec/bundler_inject_spec.rb

```diff
@@ -143,6 +143,30 @@
           end
         end
 
+        it "with a filename path in a gem_path location " do
+          with_path_based_gem("https://github.com/rubyworks/ansi", "the_gem") do |path|
+            write_bundler_d_file <<~F
+              override_gem "ansi", :path => "the_gem"
+            F
+            bundle(:update, env: { "BUNDLE_BUNDLER_INJECT__GEM_PATH" => "#{path.dirname.to_s}/"})
```

Comment:
I think the .to_s here is redundant

Reply:
I removed the interpolation, but ended up needing the `to_s` again. (blew up without it)

Reply:
Right, if interpolating you don't need the to_s, but if you removed the interpolation, then you do need it.